CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to Terms Types
+
+Thank you for considering contributing to the `terms-types` project!  🎉👍
+
+Suggestions for new types of terms of service are welcome. To ensure a smooth process, please follow the steps outlined below:
+
+## Process for adding a new terms type
+
+1. **Open a discussion**
+   - Start a new discussion with the type "New Type" in the [Discussions section](https://github.com/opentermsarchive/terms-types/discussions).
+
+2. **Clearly define the need**
+   - Explain why you believe a new terms type is needed.
```

Comment:
I feel this is not specific enough. A new terms type should always and only be needed because no existing one matches the terms that the contributor wants to track. What do you mean here?